repos_info.json

```diff
@@ -34,12 +34,12 @@
       "versioning_strategy": "python"
     },
     {
-      "name": "mit-open",
+      "name": "mit-learn",
       "repo_url": "https://github.com/mitodl/mit-open.git",
-      "ci_hash_url": "https://api.mit-open-ci.odl.mit.edu/static/hash.txt",
-      "rc_hash_url": "https://api.mitopen-rc.odl.mit.edu/static/hash.txt",
-      "prod_hash_url": "https://api.mitopen.odl.mit.edu/static/hash.txt",
-      "channel_name": "product-mit-open-next",
+      "ci_hash_url": "https://api.ci.learn.mit.edu/static/hash.txt",
+      "rc_hash_url": "https://api.rc.learn.mit.edu/static/hash.txt",
+      "prod_hash_url": "https://api.learn.mit.edu/static/hash.txt",
+      "channel_name": "product-mit-learn",
```

Comment:
Regarding production, we haven't switched yet, right? But I think that's OK? Doof will just be a little broken in terms of slacking us, but will still deploy?

Or we could do prod later.

Reply:
I don't want to have to do another PR this afternoon to fix production, I think this is just a march forward on DNS changes regardless of pace.